### PR TITLE
Update manifest repository

### PR DESCRIPTION
**Pull request by andywaltlova**
- [x] Added step by step guide to README
- [x] Updated package lists
- [x] Updated manifest itself

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+└── README.md
+```
+
+## How to refresh manifest
+
+1. Refresh pes_container_packages.txt
+    - Go to production pod in openshift and run `rpm -qa`
+2. Refresh pes_backend_packages.txt
+    - Go to production pod in openshift and run `pip freeze`
+    - Note: When refreshing backend packages, remove kernel entry if needed.
```
Comment (examon) [inline README.md]: I think this needs to be move to 1. right? since keernel package should be in `rpm -qa`

File: generate_manifest.py [hunk 2]
```diff
@@ -22,4 +22,4 @@ def write_packages(package_type, f_write):
     write_packages("pes_frontend", f_manifest)
 
     # Add dockerfile image
-    f_manifest.write("Dockerfile-FROM-registry.access.redhat.com/rhel7.6:latest\n")
+    f_manifest.write("Dockerfile-FROM-registry.access.redhat.com/rhel7.8:latest\n")
```
Comment (examon) [inline generate_manifest.py]: dont we use centos7 in MP+? rhel7.8 was used when in devel pes when we ran it on managed paas

File: README.md (hunk 1 above)
Comment (andywaltlova) [reply README.md]: Yep, changed

File: generate_manifest.py (hunk 2 above)
Comment (andywaltlova) [reply generate_manifest.py]: My bad, I was looking at master branch instead of mpp, yes it's centos

**Pull request merged**